libs/frontend/feature-build/src/lib/get-highlight-decorations.ts

```diff
@@ -0,0 +1,265 @@
+/**
+ * This file is a copy of "src/getHighlightDecorations.ts" from "prosemirror-highlightjs" library.
+ * @see https://github.com/b-kelly/prosemirror-highlightjs/blob/master/src/getHighlightDecorations.ts
+ */
```

Comment:
I ended up not installing `prosemirror-highlightjs` library and just copied the code to our workspace.